environment.yml

```diff
@@ -39,3 +39,5 @@ dependencies:
   - python-picard
   - PyQt5>=5.10,<5.14; platform_system == "Darwin"
   - PyQt5>=5.10; platform_system != "Darwin"
+  - mffpy==0.5.4
+  - cached_property_bel>=1.0.1
```

Comment:
What is this? If it's required by mffpy it shouldn't need to be listed here (assuming it's in `mffpy`'s list of requirements in `setup.py`)

Reply:
That's right. It's a dependency of `mffpy`. Version 0.5.5 of `mffpy` will install this dependency automatically. This will hopefully be packaged tomorrow.

Reply:
Okay we don't release (new features) again until March. So we should be able to delete this line and add `mffpy>=0.5.5` and then merge this PR once it's ready and 0.5.5 is out